src/proto/FunctionRpc.proto

```diff
@@ -240,6 +240,13 @@ message FunctionEnvironmentReloadRequest {
 message FunctionEnvironmentReloadResponse {
   // Status of the response
   StatusResult result = 3;
+
+  // After specialization, worker sends capabilities & metadata.
+  // A map of worker supported features/capabilities
+  map<string, string> capabilities = 2;
+
+  // Worker metadata captured for telemetry purposes
+  WorkerMetadata worker_metadata = 1;
```

Comment:
Not experienced with protobuf too much, but how will it work inserting a `1` here?

Reply:
I meant to leave a comment about this. forgot.